platform/elrond/model.go

```diff
@@ -47,6 +49,19 @@ type Transaction struct {
 	Timestamp time.Duration `json:"timestamp"`
 	Status    string        `json:"status"`
 	Fee       string        `json:"fee"`
+	GasPrice  uint64        `json:"gasPrice,omitempty"`
```

Comment:
is it possible just to use `Fee` ? if gas price and gas limit is optional

Reply:
For the moment the fields GasPrice and GasLimit cannot be removed because we use these fields to do an estimation of the transaction fee.